### ISSUE TITLE
story building

### ISSUE THREAD

**Issue by threethao** [TienDat2704/CauTot#5]
![Image](https://github.com/TienDat2704/twdf/assets/171463084/6d38f382-eff5-4bce-9eb2-1cc2db8d4829)

chị thử rà lại logic game thì không hợp lý lắm.
như biểu đồ ở trên, quái vật đi từ ngoài (chỗ làm) vào trong (backyard -> nhà -> lên tầng). suy ra, quái vật sẽ luôn phải thắng ở các LV thì mới có thể đi xuyên qua được.
=> phe bảo vệ người sẽ luôn phải thua

**Comment by threethao** [comment]
một cách chỉnh story hợp lý & deep hơn là:
+ những con pet bảo vệ người là những định kiến có sẵn trong người / một dạng defense mechanism trong psychology 
+ những con quái đi từ ngoài vào thực ra là biểu tượng của hiện thực đúng / một tiếng vọng từ bên ngoài để phá bỏ định kiến

=> phần này, có thể đến đoạn cuối thì chèn 1 chiếc ảnh hé lộ quái với hình ảnh tươi sáng hơn, chuyển từ phản diện thành chính diện

còn cách để quái breakthrough qua các LV thì chị nghĩ cứ dùng một con siêu to khổng lồ lăn vào phá vỡ tất cả defense là ổn

**Comment by TienDat2704** [comment]
Oki, vậy là cuối lv con boss khổng lồ sẽ lăn vào và chúng nó sẽ chiến thắng đúng không ạ. Để rồi mở ra lv 2 theo hướng storyline mới

**Comment by threethao** [comment]
Yes, kiểu mình để mọi người chơi thôi chứ đằng nào cũng thua ấy :v
